lib/fathead/npm_cli/cover/redirects.txt

```diff
@@ -0,0 +1,74 @@
+npm access public, access
+npm access restricted, access
```

Comment:
@pjhampton you're correct. `npm` and `npm cli` are triggers, meaning if they are at the beginning or end of any article or redirect title, it will still trigger as expected 👍 

please remove "npm" from all of these